content/en/docs/collector/deployment/gateway/index.md

```diff
@@ -198,6 +199,30 @@ The load-balancing exporter emits metrics including
 that you can use for health and performance monitoring of the OTLP endpoint
 collector.
 
+## Typical Gateway deployment architecture
+
+The following diagram shows a typical deployment architecture for a gateway
+![gateway](otel-gateway-arch.svg)
```

Comment:
Can you describe in a few words what I see on that diagram?

Reply:
Added .